js/components/widgets/image-picker.vue

```diff
@@ -133,7 +133,10 @@ export default {
             if (this.HAS_FILE_API) {
                 const data = {};
                 try {
-                    data.bbox = this.$refs.thumbnailer.bbox;
+                    let bbox = this.$refs.thumbnailer.bbox;
```

Comment:
Shoumd be a `const` instead of a `let`

Reply:
Yes :-/ Worth a new PR?

Reply:
Nop, this shouldn't have been merged: breaks on lower resolutions (non-retina) and mobile :/

Reply:
Really? :-( How? I just tested on mobile (retina) and it looks ok.

Reply:
No sorry, wrong window ;)
The PR is welcomed for the `s/let/const/`

Reply:
😌